### PR TITLE
[CI] Generate tracks summary

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: .github/bin/languages/update.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Setup the git user (required to commit anything)
+git config --global user.email "github-actions[bot]@users.noreply.github.com"
+git config --global user.name "github-actions[bot]"
+
+echo "Committing updated languages summary"
+
+# Commit the changes
+git add .
+git commit -m "[CI] Update languages summary"
+git push
```
Comment (SaschaMann) [inline .github/bin/languages/update.sh:28]: Pushing straight to master will have the same issue that was reported on Slack earlier: https://github.com/exercism/v3/runs/798972403?check_suite_focus=true

A workaround would be pushing it to a branch and then having an auto merge account merge it (I think @iHiD mentioned he created a machine account for Exercism?)

Comment (ErikSchierboom) [reply .github/bin/languages/update.sh:28]: Indeed. I'll have to fix this

File: .github/workflows/languages-summary.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Languages summary
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'languages/*/config.json'
+      - 'languages/*/exercises/**'
```
Comment (SaschaMann) [inline .github/workflows/languages-summary.yml:9]: I think this might be missing a trailing *? Would have to check the other workflows to be sure though

Comment (ErikSchierboom) [reply .github/workflows/languages-summary.yml:9]: I've borrowed this from the `julia-exercise-ci.yml` workflow and it seems to work for the stories one too.

**Pull request merged**